src/hlstrafe.cpp

```diff
@@ -2127,6 +2166,14 @@ namespace HLStrafe
 			VecScale<float, 2>(playerCopy.Velocity, (100.0 - (playerCopy.StaminaTime / 1000.0) * 19.0) / 100.0, playerCopy.Velocity);
 
 		CheckVelocity(playerCopy, vars);
+
+		if (curState.Parameters.Type == HLTAS::ConstraintsType::LOOK_AT) {
+			auto playerCopy2 = PlayerData(playerCopy); // trace origin after Strafe() before it happens
+			Strafe(playerCopy2, vars, postype, frame, out, reduceWishspeed, strafeButtons, useGivenButtons, true, curState, traceFunc, pointContentsFunc, version, out.fractions, out.normalzs);
+			UpdateLookAtViewangle(playerCopy2, curState);
+			out.Pitch = curState.TargetYawLookAtPitch;
```

Comment:
Hmm, I'm not sure how I feel about look_at overriding pitch set in frame bulk. I think it would make more sense for pitch set in frame bulk to override look_at for this frame bulk instead. What do you think?

Reply:
reasonable